storage/innobase/log/log0recv.cc

```diff
@@ -1852,7 +1868,39 @@ recv_parse_or_apply_log_rec_body(
 	        Otherwise, redo will not find the key to decrypt
 		the data pages. */
 		if (page_no == 0 && !is_system_tablespace(space_id)) {
-			if (fil_write_encryption_parse(ptr, end_ptr,
+			byte* ptr_copy = ptr;
+			//increase by offset
+			ptr_copy += 2;
+			ulint len = mach_read_from_2(ptr_copy);
+			ptr_copy += 2;
+			if (end_ptr < ptr_copy + len) {
+				return NULL;
+			}
+
+                        /* With Percona Server, Tables created with
```

Comment:
comment starts at wrong offset (column)

Reply:
ideally clang-format should have moved this. Can you run clang-format on this file?